apps/fz_http/priv/static/browserconfig.xml

```diff
@@ -2,6 +2,7 @@
 <browserconfig>
     <msapplication>
         <tile>
+            <!-- TODO: we should render this file in Elixir if we want dynamic path work for browserconfig -->
```

Comment:
This is a TODO I will tackle in case PR looks good to merge.

Reply:
Good catch!